core/src/test/java/nl/nn/adapterframework/configuration/ConfigurationUtilsTest.java

```diff
@@ -499,14 +498,12 @@ public void stub4testtoolFullAdapter() throws Exception {
 	
 	private void stub4testtoolTest(String baseDirectory, boolean disableValidators) throws Exception {
 		Map<String, Object> parameters = new Hashtable<String, Object>();
-		// Parameter disableValidators has been used to test the impact of
-		// validators on memory usage.
-		parameters.put("disableValidators", disableValidators);
+		parameters.put(ConfigurationUtils.STUB4TESTTOOL_XSLT_VALIDATORS_PARAM, disableValidators);
```

Comment:
In een unit test moet je niet de constanten uit de class under test gebruiken, maar in de test class gedefinieerde constanten. Daarmee zorg je er voor dat als iemand per ongeluk de waarde van deze constanten verandert, dat dat opgemerkt wordt doordat de test faalt.